### PR TITLE
Add 6.0.1 CHANGELOG entry

### PR DESCRIPTION
**Pull request by byucesoy**
Couple fixes and improvements to existing behavior.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+### citus v6.0.1 (November 29, 2016) ###
+
+* Fixes a bug preventing DML queries while creating colocated table
```
Comment (jasonmp85) [inline CHANGELOG.md]: Reads better with "during": _Fixes a bug preventing DML queries during colocated table creation_

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+* Fixes a bug causing incorrectly passing down NULL parameters as text
```
Comment (jasonmp85) [inline CHANGELOG.md]: Not syntactic. Maybe _Fixes a bug that caused NULL parameters to be incorrectly passed as text_?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+
+* Fixes a bug causing failures during pg_upgrade
```
Comment (jasonmp85) [inline CHANGELOG.md]: Move this to the top (most impactful or important `CHANGELOG` should go to the top).

**Comment by byucesoy** [comment]
I addressed your comments regarding the commit summaries.

**Pull request merged**